Ability to force a theme on Windows (#1666)
rust-windowing/winit: CHANGELOG.md

# Unreleased

> - ***Breaking:** On Windows, Renamed `WindowBuilderExtWindows`'s is_dark_mode` to `theme`.

Reviewer: ```suggestion
- **Breaking:** On Windows, Renamed `WindowBuilderExtWindows`'s `is_dark_mode` to `theme`.
```
(see https://github.com/rust-windowing/winit/blob/1ef38f04127d3399b792aefcbcb233cf390ab5d7/CHANGELOG.md)

# Unreleased

- On Android, calling `WindowEvent::Focused` now works properly instead of always returning false. 

> - On Windows, add `WindowBuilderExtWindows::with_theme(Theme::)` to set a preferred theme.

Reviewer 2: Should append into the changelog, and not insert between. Also, there's no need to provide argument in changelog, just a name.

```suggestion
- On Windows, add `WindowBuilderExtWindows::with_theme` to set a preferred theme.
```

Author: 👍 commited

- On Web, prevent the webpage from scrolling when the user is focused on a winit canvas
- On Web, calling `window.set_cursor_icon` no longer breaks HiDPI scaling
- On Windows, drag and drop is now optional and must be enabled with `WindowBuilderExtWindows::with_drag_and_drop(true)`.

> - On Windows, force themes now with `WindowBuilderExtWindows::with_theme(Theme::)`.

Reviewer 2: The location is wrong, please rebase that PR properly.

I'd also word it like

```suggestion
- On Windows, add `WindowBuilderExtWindows::with_theme(Theme::)` to force specific theme.
```
